msg/tools/uorb_to_ros_rtps_ids.py

```diff
@@ -45,7 +46,7 @@
     '''
 __credits__ = ['Nuno Marques <nuno.marques@dronesolution.io>']
 __license__ = 'BSD-3-Clause'
-__version__ = '0.1.0'
+__version__ = subprocess.check_output('git describe --abbrev=0', shell=True).strip().decode()
```

Comment:
Please don't do that, it's an unnecessarily expensive process execution.

Reply:
Ok sure. Will remove it